kobocat/onadata/apps/api/viewsets/xform_submission_api.py

```diff
@@ -210,12 +210,15 @@ def create(self, request, *args, **kwargs):
         context = self.get_serializer_context()
         serializer = SubmissionSerializer(instance, context=context)
 
+        print("XFormSubmissionApi.create")
+        print(instance.date_created.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z')
+
         return Response({
             'formid': instance.xform.id_string,
             'encrypted': instance.xform.encrypted,
             'instanceID': u'uuid:%s' % instance.uuid,
-            'submissionDate': instance.date_created.isoformat(),
-            'markedAsCompleteDate': instance.date_modified.isoformat()},
+            'submissionDate': instance.date_created.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z',  # TODO MAKE ME THE SAME AS JS ISOSTRING!
```

Comment:
@ChasNelson1990 this "todo" comment is an artifact, right, because what you're doing here is exaclty that, right?